natlas-agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6.0 as build
+FROM python:3.6 as build
```

Comment:
I wonder if we can't use python:3.7 here, since we also use it for the server. It'd be good if both halves of the project targeted the same python version.

Reply:
Yep. I noticed that but didn't want to make that change at the same time since it would have been riskier. Let's tackle that while we're doing py 3.8 bump.